backend-app/utils/swagger/mergeYamlFiles.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+const fs = require('fs');
+const YAML = require('yamljs');
+
+/**
+ * This file contains the function that merges all the YAML files in a directory
```

Comment:
it would be better if you keep only this line to know what the function do , and only nessesery comments not commenting every line  and also : "This file contains the function that merges all the YAML files in a directory" => "Merges all the YAML files in a directory"

Reply:
i will remove those unnecessary comments.